sdk/scheduler/src/test/java/com/mesosphere/sdk/specification/yaml/YamlToInternalMappersTest.java

```diff
@@ -0,0 +1,44 @@
+package com.mesosphere.sdk.specification.yaml;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * This class tests {@link YAMLToInternalMappers}.
+ */
+public class YamlToInternalMappersTest {
+    private static final String firstUriStr = "https://github.com/mesosphere/dcos-commons/blob/master/README.md";
+    private static final String secondUriStr = "https://github.com/mesosphere/dcos-commons/blob/master/docs/generate.sh";
+    private static final String multipleSpacedUriStr = firstUriStr + ", " + secondUriStr;
+    private static final String multipleNoSpaceUriStr = firstUriStr + "," + secondUriStr;
+
+    @Test(expected = URISyntaxException.class)
+    public void testEmptyUriString() throws URISyntaxException {
+        YAMLToInternalMappers.getUris("");
+    }
+
+    @Test
+    public void testSeparateSingleUriString() throws URISyntaxException {
+        URI uri = YAMLToInternalMappers.getUris(firstUriStr).stream().findAny().get();
+        Assert.assertEquals(firstUriStr, uri.toString());
```

Comment:
Since this is a new file, I also just want to ask what the view is on the Hamcrest matchers? We could write this as:
```java
@Test
    public void testSeparateSingleUriString() throws URISyntaxException {
        final Collection<URI> uris = YAMLToInternalMappers.getUris(firstUriStr);
        assertThat(uris, equalTo(ImmutableList.of(new URI(firstUriStr))));
    }
```

and one of the following as:
```java
    @Test
    public void testSeparateCommaWithSpace() throws URISyntaxException {
        final Collection<URI> uris = YAMLToInternalMappers.getUris(multipleSpacedUriStr);
        assertThat(uris, equalTo(ImmutableList.of(new URI(firstUriStr), new URI(secondUriStr))));
    }
```